r/tests/testthat/test-dplyr-lubridate.R

```diff
@@ -32,97 +32,132 @@ if (tolower(Sys.info()[["sysname"]]) == "windows") {
 } else {
   test_date <- as.POSIXct("2017-01-01 00:00:12.3456789", tz = "Pacific/Marquesas")
 }
-test_df <- tibble::tibble(date = test_date)
-test_date_df <- tibble::tibble(date = as.Date(as.character(test_date)))
 
-test_that("extract year from date32 objects", {
-  expect_equivalent(
-    test_date_df %>%
-      mutate(x = year(date)) %>%
-      collect() %>%
-      select(x),
-    test_df %>%
-      mutate(x = year(date)) %>%
-      collect() %>%
-      select(x)
+test_df <- tibble::tibble(
+  datetime = test_date,
+  date = as.Date("2021-09-09")
```

Comment:
Wouldn't hurt to confirm the missing data behavior is consistent.

```suggestion
  datetime = c(test_date, NA),
  date = c(as.Date("2021-09-09"), NA)
```